samples/rag-aisearch/README.md

```diff
@@ -73,3 +73,27 @@ Semantic Search binding for Azure AI Search expects/creates a very specific inde
 ### Vector Search Configuration
 
 Algorithm - HnswAlgorithmConfiguration
+
+### Use of API Key
+
+Azure AI Search offers key-based authentication that you can use on connections to your search service. [More information](https://learn.microsoft.com/azure/search/search-security-api-keys)
+
+1. Use of API Key is optional and managed identities are recommended way for authentication.
```

Comment:
This bullet should be moved into the above paragraph since it's not related to any instructions on *how* to use API keys.